app/helpers/application_helper.rb

```diff
@@ -14,4 +14,10 @@ def i18n_pluralize(word)
   def i18n_error_count(count)
     I18n.locale == :ja ? "#{count}件の#{t('views.common.error')}" : pluralize(count, t('views.common.error'))
   end
+
+  def name_or_email_or_deleted_user(object)
```

Comment:
```suggestion
  def name_or_email_or_deleted_user(record)
```

ActiveRecordのオブジェクトだと思うので、recordの方が少し具体性が上がっていいかもしれません

あとメソッド名がちょっと具体的すぎるのでもう少し抽象度を上げた方がいいですね
https://bootcamp.fjord.jp/articles/78

Reply:
実を言うと、ここの引数名とメソッド名はどちらも悩みました（特にメソッド名は自分でも野暮ったく感じた）
「ユーザー名表示を変更している」という役割から、`change_display_name`にしましたが、あまり自信がありません。